### PR TITLE
Fixes #22337 - added PXEGrub1/2 auto-association settings

### PR DESCRIPTION
**Pull request by lzap**
Full disclosure, I haven't tested this.

The new templates are really named:

* Kickstart default PXEGrub
* Kickstart default PXEGrub2

So it *should* just wrok :-)

### CODE REVIEW

**Comment by lzap** [comment]
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1385796

**Comment by ares** [comment]
LGTM

File: app/helpers/katello/concerns/settings_helper_extensions.rb [hunk 1]
```diff
@@ -12,6 +12,8 @@ def value(setting)
             'katello_default_provision',
             'katello_default_ptable',
             'katello_default_PXELinux',
+            'katello_default_PXEGrub',
+            'katello_default_PXEGrub2',
```
Comment (jlsherrill) [inline app/helpers/katello/concerns/settings_helper_extensions.rb]: 00:04:49.998 /var/lib/workspace/workspace/test_katello_pull_request/slave/fast/plugin/app/helpers/katello/concerns/settings_helper_extensions.rb:7:9: C: Metrics/MethodLength: Method has too many lines. [35/30]
00:04:49.998         def value(setting) ...

I think this needs a rubocop override

Comment (lzap) [reply app/helpers/katello/concerns/settings_helper_extensions.rb]: Do you prefer local or global?

Comment (lzap) [reply app/helpers/katello/concerns/settings_helper_extensions.rb]: Amended local comment disable.

Comment (jlsherrill) [reply app/helpers/katello/concerns/settings_helper_extensions.rb]: Yep, local. Thanks! 👍

**Comment by jlsherrill** [comment]
[test rails-5]

**Pull request merged**